python/rmm/rmm/_lib/device_buffer.pyx

```diff
@@ -149,7 +149,7 @@ cdef class DeviceBuffer:
         device : optional
             The CUDA device to which to prefetch the memory for this buffer.
             Defaults to the current CUDA device. To prefetch to the CPU, pass
-            `~cuda.cudart.cudaCpuDeviceId` as the device.
+            :py:`~cuda.cudart.cudaCpuDeviceId` as the device.
```

Comment:
```suggestion
            :py:attr:`~cuda.cudart.cudaCpuDeviceId` as the device.
```
I think.

Reply:
In theory the extra qualifier is nice to have, but given that the CI failed it might not be a bad idea to give it a try.